### PR TITLE
Blog addition

### PR DESCRIPTION
**Pull request by aesherman**
## I ensure I have done the following steps

- [x] Star the repository :star:
- [x] Followed the [author](https://github.com/piyush97):white_check_mark:
- [x] Tested the code properly :white_check_mark:
- [x] Added the required feature :white_check_mark:

### CODE REVIEW

File: content/blog/testing-RN-with-Appium/index.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+slug: 'testing-RN-with-Appium'
+title: 'How we test our React-Native application with Appium'
+date: '2020-10-02'
+author: 'Andrew Sherman'
+published: true
+description: 'Software testing made simple'
```
Comment (piyush97) [inline content/blog/testing-RN-with-Appium/index.mdx:7]: Image is a required param.

**Review by piyush97 — CHANGES_REQUESTED** [review]
Image is a required param. Please fix the build!

File: content/blog/testing-RN-with-Appium/index.mdx (hunk 1 above)
Comment (aesherman) [reply content/blog/testing-RN-with-Appium/index.mdx:7]: Yo if you want a change that's fine, request a change.  Don't just put a comment in and then close the PR.

Comment (aesherman) [reply content/blog/testing-RN-with-Appium/index.mdx:7]: Also pretty sketchy to mark your repo as participating in Hacktoberfest, require people to star and follow you, then remove the repo from participating in hacktoberfest.

**Pull request closed without merging**